content/lessons/chapter-2/mining-1.tsx

```diff
@@ -11,6 +11,7 @@ import {
   TitleCard,
   HashFrequency,
   StartButton,
+  BoxDisplayer,
```

Comment:
``` suggestion
```

Reply:
Addressed in 8852c948c97a12026a90871b9b7c7208e8ebd52b
Thanks for reviewing!